src/sandbox-posix.c

```diff
@@ -825,6 +825,8 @@ int bxfi_exec(bxf_instance **out, bxf_sandbox *sandbox,
     argv[argc++] = NULL;
 
     execve(fullpath, argv, env);
+    free(env);
+    free(instance);
```

Comment:
I disagree with freeing things here, for two reasons.

The first is that we only get to this point when things go wrong. We're not supposed to hit that case in the vast majority of cases, and since we're promptly aborting the program, the OS is going to claim everything anyway, so I feel that this is not worth the cleanup.

The second is about fork-safety. Pretty much only async-signal-safe functions can be called after `fork()`, and `free()` isn't one of them. It's generally fine, but I'd like to let users spawn workers reliably in a multithreaded context.

Re-reading the code also made me realize that calling abort() and dupenv() here is bogus. I'll have to address that later.